plugins/sql-language/src/main/java/org/polypheny/db/sql/language/ddl/altertable/SqlAlterSourceTableAddColumn.java

```diff
@@ -110,35 +106,25 @@ public void unparse( SqlWriter writer, int leftPrec, int rightPrec ) {
 
 
     @Override
-    public void execute( Context context, Statement statement, QueryParameters parameters ) {
-        CatalogTable catalogTable = getCatalogTable( context, table );
+    public void execute( Context context, Statement statement, ParsedQueryContext parsedQueryContext ) {
+        LogicalTable logicalTable = getTableFailOnEmpty( context, table );
```

Comment:
Add a check whether the found schema object actually is a table and not, for instance, a view.

Reply:
This is already done on the line below `if ( logicalTable.entityType != EntityType.SOURCE ) {..`

Reply:
Where? Am I blind?
<img width="1003" alt="image" src="https://github.com/polypheny/Polypheny-DB/assets/1682115/45680c53-d18a-47f7-aab3-3dcf90b240df">